connectivity/FEATURE_BLE/source/cordio/TESTS/cordio_hci/driver/main.cpp

```diff
@@ -37,8 +37,9 @@
 
```

Comment:
The commit message "Greenta: Remove deprecated APIs and use chrono " should be "Greentea: Remove deprecated APIs and use chrono"